app/controllers/trip_families_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TripFamiliesController < ApplicationController
+  def index
+    @trip_families = trip.trip_families.all
+  end
+
+  def show
+    @trip_family = trip.trip_families.find(params[:id])
+
+    respond_with @trip_family, location: -> { trip_family_path }
+  end
+
+  def new
+    @trip_family = trip.trip_families.new
+  end
+
+  def create
+    @trip_family = @trip.trip_families.create(trip_family_params)
+
+    respond_with @trip, @trip_family,
```

Comment:
A suggestion to change this to be: `[trip, @trip_family]` to make it clearer.
Or alternatively, use the spelled out path `trip_trip_family_path(trip, @trip_family)`

Reply:
Done in sha c8fc6f2